test/cpp/end2end/xds/xds_outlier_detection_end2end_test.cc

```diff
@@ -1335,8 +1335,8 @@ int main(int argc, char** argv) {
   // CoreConfiguration::BuildSpecialConfiguration() instead.
   gpr_setenv("GRPC_EXPERIMENTAL_ENABLE_OUTLIER_DETECTION", "true");
```

Comment:
Can we now address the TODO immediately above so that we don't need to mess with the env vars here?

Reply:
I tried and got confused: probably better that someone who knows that code go through in a separate pass.